Available at your primary URL 
==> 
==> ///////////////////////////////////////////////////////////
/opt/render/project/src/backend/node_modules/pg-protocol/dist/parser.js:285
        const message = name === 'notice' ? new messages_1.NoticeMessage(length, messageValue) : new messages_1.DatabaseError(messageValue, length, name);
                                                                                                 ^
error: syntax error at end of input
    at Parser.parseErrorMessage (/opt/render/project/src/backend/node_modules/pg-protocol/dist/parser.js:285:98)
    at Parser.handlePacket (/opt/render/project/src/backend/node_modules/pg-protocol/dist/parser.js:122:29)
    at Parser.parse (/opt/render/project/src/backend/node_modules/pg-protocol/dist/parser.js:35:38)
    at TLSSocket.<anonymous> (/opt/render/project/src/backend/node_modules/pg-protocol/dist/index.js:11:42)
    at TLSSocket.emit (node:events:518:28)
    at addChunk (node:internal/streams/readable:561:12)
    at readableAddChunkPushByteMode (node:internal/streams/readable:512:3)
    at Readable.push (node:internal/streams/readable:392:5)
    at TLSWrap.onStreamRead (node:internal/stream_base_commons:189:23) {
  length: 91,
  severity: 'ERROR',
  code: '42601',
  detail: undefined,
  hint: undefined,
  position: '44',
  internalPosition: undefined,
  internalQuery: undefined,
  where: undefined,
  schema: undefined,
  table: undefined,
  column: undefined,
  dataType: undefined,
  constraint: undefined,
  file: 'scan.l',
  line: '1236',
  routine: 'scanner_yyerror'
}
Node.js v22.16.0
